exchanges/order/order_test.go

```diff
@@ -785,6 +784,7 @@ func TestStringToOrderType(t *testing.T) {
 	for i := range cases {
 		testData := &cases[i]
 		t.Run(testData.in, func(t *testing.T) {
+			t.Parallel()
```

Comment:
suggestion: Seeing as there is not much needed to be done and t.Run doesn't need to be called but can stay as is.